net-libs/libssh2/libssh2-1.10.0.ebuild

```diff
@@ -30,6 +30,10 @@ DEPEND="
 	${RDEPEND}
 "
 
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.10.0-libressl.patch #903001
```

Comment:
If it's upstreamed, it's going to be part of the next release, right? So it'd be better to use `${P}-` here, right?

Reply:
I changed it, but they have release branches for `1.8.1` and `1.8.x` so I am unsure if this will be in `1.10.1` or in `1.11.0`. The last release (`1.10.0`) was on Aug 29, 2021 so I am also unsure when the next release will be.